doc/release-notes/rl-2009.adoc

```diff
@@ -43,3 +43,9 @@ $ nix-shell '<home-manager>' -A install
 
 will automatically include these options, when necessary.
 --
+
+* Git's `smtpEncryption` option now only is set to `tls` if
+<<opt-accounts.email.accounts.<name>.smtp.tls.enable>> AND
```

Comment:
Try
```suggestion
<<opt-accounts.email.accounts.\_name_.smtp.tls.enable>> AND
```
and similar for the other references.

The anchor in https://rycee.gitlab.io/home-manager/options.html#opt-accounts.email.accounts._name_.smtp.tls.enable shows what the reference should be.

The first `_` needs to be escaped using `\_` to avoid expansion into an `<emphasis>` element.

Reply:
Ah that makes sense! Thanks! :bow: